bindata/network/ovn-kubernetes/managed/monitor.yaml

```diff
@@ -0,0 +1,160 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: ovnkube-master
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-ovn-master-metrics
+  namespace: {{.HostedClusterNamespace}}
+  clusterName: {{.ManagementClusterName}}
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+    scheme: https
+    bearerTokenSecret:
+      key: ""
+    tlsConfig:
+      ca:
+        configMap:
+          key: service-ca.crt
+          name: openshift-service-ca.crt
+      cert:
+        secret:
+          key: tls.crt
+          name: ovn-master-metrics-cert
+      keySecret:
+        key: tls.key
+        name: ovn-master-metrics-cert
+      serverName: ovn-kubernetes-master.{{.HostedClusterNamespace}}.svc
+    metricRelabelings:
+    - action: replace
+      replacement: {{.ClusterID}}
+      targetLabel: {{.ClusterIDLabel}}
+    relabelings:
+    - action: replace
+      replacement: {{.ClusterID}}
+      targetLabel: {{.ClusterIDLabel}}
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - {{.HostedClusterNamespace}}
+  selector:
+    matchLabels:
+      app: ovnkube-master
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ovnkube-master
+  name: ovn-kubernetes-master
+  namespace: {{.HostedClusterNamespace}}
+  clusterName: {{.ManagementClusterName}}
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: ovn-master-metrics-cert
+spec:
+  selector:
+    app: ovnkube-master
+  clusterIP: None
+  publishNotReadyAddresses: true
+  ports:
+  - name: metrics
+    port: 9102
+    protocol: TCP
+    targetPort: 9102
+  sessionAffinity: None
+  type: ClusterIP
+---
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: ovnkube-node
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-ovn-node
+  namespace: openshift-ovn-kubernetes
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: ovn-kubernetes-node.openshift-ovn-kubernetes.svc
+  - interval: 30s
+    port: ovn-metrics
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: ovn-kubernetes-node.openshift-ovn-kubernetes.svc
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - openshift-ovn-kubernetes
+  selector:
+    matchLabels:
+      app: ovnkube-node
+---
+apiVersion: v1
```

Comment:
Does it make sense to move the ovn-kube node monitoring bits in to `/shared`? Or do we expect them to be different?

Reply:
sure, the ovnkube-node monitoring manifest are shared, not difference.
Moved the ovnkube-node service and serviceMonitor to common folder.